bump_version.sh

```diff
@@ -37,9 +46,12 @@ else
       # A temp file is used to provide compatability with macOS development
       # as a result of macOS using the BSD version of sed
       tmp_file=/tmp/version.$$
+      tmp_flake=/tmp/flake.tmp
       sed "s/$old_version_regex/$new_version/" $VERSION_FILE > $tmp_file
       mv $tmp_file $VERSION_FILE
-      git add $VERSION_FILE
+      sed "s/$old_version_regex/$new_version/" $FLAKE_FILE > $tmp_flake
+      mv $tmp_flake $FLAKE_FILE
+      git add $FLAKE_FILE $FLAKE_LOCK_FILE $VERSION_FILE
```

Comment:
Since we don't touch the lock file here I don't want to confuse a future reader with why we're trying to add it now.

```suggestion
      git add $FLAKE_FILE $VERSION_FILE
```

Reply:
Rushed on my part